### ISSUE TITLE
The `alt` text of logo in emails should always be present for a11y.

### ISSUE THREAD

**Issue by jimleroyer** [cds-snc/notification-planning#32]
### Task Description

The email currently generated are not accessible on the logo image because the `alt` attribute is not properly populated when the brand text is defined.

### Task Outcome

The email is more accessible with an `alt` attribute that is always filled.

### Acceptance Criteria

- [ ] Change the `alt` attribute to be filled by either the brand name if the brand text is missing, else the brand text when present.

### Additional Notes

This might not be the ideal solution. Better would be to have a dedicated text box in the admin UI for accessibility purpose. It seems the brand text was initially meant for services to have a tiny logo along with a text name beside it. I need to confirm with @willeybryan what would be the best road for this feature as he might have more insights on the branding logo options.

**Comment by jimleroyer** [comment]
The PR would be ready if we go forward with it:
https://github.com/cds-snc/notification-utils/pull/105

**Comment by jimleroyer** [comment]
I spoke to Bryan to know what's the intention of brand text. As far as he knows, this isn't used by most of our clients as him and Antoine did not fill these. I extracted the services that has a brand text defined from the database. Here are the results:

```
Logo - Upload Demo	Canadian Digital Service
CDS	Canadian Digital Service - Service numérique canadien
Elections test	Elections Canada
Logo de Normes d’accessibilité Canada	Normes d'accessibilité Canada - Accessibility Standards Canada
Accessibility Standards Canada Logo	Accessibility Standards Canada - Normes d'accessibilité Canada
Jimmy Brand Logo	Jimmy Brand
```

Most of these are tests. Accessibility standards is the one service that brought forward the issue and hence we set the brand text on it. The only one I am unsure about actually is the CDS service having a brand text. It might be anormal and I will test further to see how our emails are rendered.

Regarding brand text as a feature, it is my understanding that it is meant to provide a logo subtitle and make the logo smaller. Bryan mentioned to me it is unlikely this was the desired outcome for users we have that we might have filled this with.

I see two outcome with this.

1. A quick fix ([link to PR implementing the API / utils side](https://github.com/cds-snc/notification-utils/pull/105)) where we redirect the brand text feature to be an accessibility feature meant for the `alt` accessibility text. We would lose the other feature which seems unused by redirecting the intent.
2. A medium fix where we add a new box meant for accessibility text in the admin brand configuration screen. This would require a new column in the database and to transport the information from the admin to the API. It might not be that much effort and would be a proper fix (but longer than the quick fix for sure).
